src/models/Settings.js

```diff
@@ -49,6 +49,7 @@ function (Okta, Q, Errors, BrowserFeatures, Util, Logger, OAuth2Util, config) {
       'stateToken': ['string', false, undefined],
       'username' : ['string', false],
       'signOutLink': ['string', false],
+      'consentCancelLink': ['string', false],
```

Comment:
not sure this should be on the top level config since it depends on ```features.consent```. Maybe it will make more sense to create a 'consent' object.
```
consent: {
    cancel: 'link/to/cancel'
}
```
@rchild-okta thoughts about this? I would like your opinion for this new config